docs-developer/requirements.txt

```diff
@@ -0,0 +1,28 @@
+Babel==2.14.0
+certifi==2024.2.2
+charset-normalizer==3.3.2
+click==8.1.7
+colorama==0.4.6
+ghp-import==2.1.0
+idna==3.6
+Jinja2==3.1.3
+Markdown==3.5.2
+MarkupSafe==2.1.5
+mergedeep==1.3.4
+mkdocs==1.5.3
+mkdocs-material==9.5.10
```

Comment:
If we want to update the version of mkdocs, I assume we need to bump `mkdocs-*` versions, but when it comes to the rest of dependencies here - how do we figure out which ones need updating as well, and to which versions? If it was copy-pasted from mkdocs documentation/examples or you somehow generated it, I think it's worth mentioning it in the README while you still remember :)

Reply:
This file was generated by running `pip freeze` (via the handy task `./gradlew :docs-developer:pipPrintRequirements`).

I've forgotten how Python works so I'm not sure how to document the correct procedure for updating Python dependencies :)